vdb/lib/__init__.py

```diff
@@ -465,12 +520,14 @@ class VulnerabilityOccurrence:
     type: str
     severity: Severity
     cvss_score: str
+    cvss_v3: CvssV3
     package_issue: PackageIssue
     short_description: str
     long_description: str
     related_urls: list
     effective_severity: Severity
     matched_by: str
+    vdetails: VulnerabilityDetail
```

Comment:
VulnerabilityDetail includes VulnerabilityLocation as a property fixed_location. Can we retrieve the vulnerability detail in a different way?

Reply:
@prabhu CSAF requires a product status, so either fixed location or affected versions must be listed (so I would want the min affected versions). Package is also needed to include any scoring, as a list of affected packages is required. The cpe may be something I want to add as well, but I haven't addressed it yet. If you prefer, I can add these as discrete properties of a VulnerabilityOccurrence, but they are needed to be able to generate a valid CSAF.

Reply:
Cool. Let's add them as discrete properties so that we don't create any cycles.

Reply:
@prabhu I'm not sure what you mean by cycles, but I am happy to extract to discrete properties.

Reply:
VulnerabilityDetail could have VulnerabilityLocation as a property.